wagtail/wagtailimages/views/serve.py

```diff
@@ -36,6 +37,15 @@ def verify_signature(signature, image_id, filter_spec, key=None):
     return signature == generate_signature(image_id, filter_spec, key=key)
 
 
+def generate_image_url(image, filter_spec, key=None):
+    signature = generate_signature(image.id, filter_spec, key)
+    url = reverse('wagtailimages_serve',
```

Comment:
Perhaps it would be useful to make the URL name configurable so a custom serve view can be used?

``{% image_url 'wagtailimages_serve' myimage fill-100x100 %}``

I think this also makes it more clearer that it's going to use the serve view rather than behaving like the ``{% image %}`` tag.